tutorials/javaFxPart3.md

```diff
@@ -67,7 +69,7 @@ Run the program and give it a whirl!
 
 At first glance everything appears to work perfectly. However, when the `VBox` stretches beyond the confines of the `ScrollPane`, the `ScrollPane` does not scroll down automatically as expected. We can remedy this by attaching a handler on the `VBox` to react to its own size changing and scrolling the `ScrollPane` down.
 
-Update the `start` method as shown below.
+Update the `Duke.start` method as shown below.
```

Comment:
Is `Duke.start` a static method? If not, use `Duke#start` instead.

Reply:
Upon another chech, yes, Duke.start is a static method. Thank you for the reminder!

Reply:
> Upon another chech, yes, Duke.start is a static method.

@laney0808 Are you sure?